### PR TITLE
Blocktime rescale

### PR DESCRIPTION
**Pull request by magnetto90**

### CODE REVIEW

File: contracts/OverlayV1Market.sol [hunk 1]
```diff
@@ -620,7 +621,7 @@ contract OverlayV1Market is IOverlayV1Market, Pausable {
     /// @dev bound = macroWindowInBlocks * reserveInOv * 2 * delta
     function backRunBound(Oracle.Data memory data) public view returns (uint256) {
         uint256 averageBlockTime = params.get(Risk.Parameters.AverageBlockTime);
-        uint256 window = (data.macroWindow * ONE) / averageBlockTime;
+        uint256 window = (data.macroWindow * ONE * TO_MS) / averageBlockTime;
         uint256 delta = params.get(Risk.Parameters.Delta);
         return delta.mulDown(data.reserveOverMicroWindow).mulDown(window).mulDown(2 * ONE);
     }
```
[the remark this answers is not in the document]
Comment (magnetto90) [reply contracts/OverlayV1Market.sol:627]: Agree

[the remark this answers is not in the document]
Comment (TomasCImach) [reply contracts/OverlayV1Market.sol:627]: > `window` is computed by dividing `macroWindow / averageBlockTime`. If we set `averageBlockTime` to be less than 1 [second], `window` will end up being greater than `macroWindow`, and the upper bound will also be higher.

The `window` variable should be the amount of blocks in a macroWindow time.
On the original formula, we were calculating `window [blocks] = macroWindow [seconds] / averageBlockTime [seconds/block]`
In the change proposal we're doing: `window [blocks] = macroWindow [seconds] * TO_MS [milliseconds/seconds] / averageBlockTime [milliseconds/block]`

The window variable should be the same if we have averageBlockTime = 1 [second] in the first formula, or averageBlockTime = 1000 [milliseconds] in the second formula.

[the remark this answers is not in the document]
Comment (magnetto90) [reply contracts/OverlayV1Market.sol:627]: This function is used only when a `hasReserve` is True, right?

**Review by TomasCImach — APPROVED** [review]

File: contracts/OverlayV1Market.sol (hunk 1 above)
[the remark this answers is not in the document]
Comment (TomasCImach) [reply contracts/OverlayV1Market.sol:627]: While the first message from @EperezOk is true (window can be greater than macroWindow, and then backRunBound would be greater than before), we can set the avgBlockTime to 1000 [milliseconds] and have the same result as before the change (using avgBlockTime = 1 [second]), with the advantage that on the future we can set a lower avgBlockTime that meets L2 realities (sub-second block times).

**Pull request merged**